crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```
@@ -50,6 +50,9 @@ spec:
       username_claim: "sub"
     openshift:
       client_id_prefix: "kiali"
+      redirect_url: ""
+      server_prefix: ""
```

Under most circumstances, the values in this example file should show the default values (people refer to this file to see what the defaults are). Based on the `defaults/main.yml` file that this PR is modifying, this should be:
```suggestion
      server_prefix: "https://kubernetes.default.svc/"
```

---

So should we document here what is hardwired in the code for defaults, when the user does not set the value? https://github.com/kiali/kiali/pull/5021/commits/3ad3ff92122e0feae44ea33a388522fa359c03e0#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R532

---

In short, these config/config.go settings are what our unit tests see :) That's why we set them here.

---

Makes sense 😄